src/mcu.cpp

```diff
@@ -1354,6 +1358,28 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (pname != NULL) {
+        int portNo = 0;
+        int length = MIDI_GetMidiInDevices(NULL);
+        auto devices = new char[length];
+        length = MIDI_GetMidiInDevices(devices);
+        auto start = devices;
+        auto end = devices + length;
+        while (start != end) {
+            int len = strlen(start);
+            if (!strcmp(start, pname)) {
```

Comment:
IMHO, it's better to use case insensitive check here (_stricmp on MSVC, strcasecmp on POSIX).

Reply:
ok